File: backend/backend/Model/Question.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace backend.Model
{
    [Table("Question")]
    public class Question
    {
        [Key]
        public Guid Id { get; set; }
        
        [Required]
        public string Text { get; set; }

        [JsonIgnore]
        [ForeignKey("QuizId")]
        public virtual Quiz? Quiz { get; set; }

        public virtual ICollection<Answer>? Answers { get; set; }
    }
}

File: frontend/scripts/parseSubjectPdf/Model/Question.py
from Model.Answer import Answer


class Question:
    def __init__(self, question: str, answers=[]) -> None:
        self._question = question
        self._answers = answers

    def get_question(self):
        return self._question

    def get_answers(self):
        return self._answers

    def add_answer(self, answer: Answer):
        self._answers.append(answer)
